go.mod

```diff
@@ -9,39 +9,31 @@ require (
 	github.com/cosmos/ledger-cosmos-go v0.10.3
 	github.com/fortytw2/leaktest v1.3.0 // indirect
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
-	github.com/gogo/protobuf v1.1.1
+	github.com/gogo/protobuf v1.2.1
```

Comment:
Do we want this version bump?

Reply:
This was indirect and not an implicit/direct update.